dp/cloud/python/magma/db_service/models.py

```diff
@@ -200,12 +200,12 @@ class DBCbsd(Base):
     state_id = Column(
         Integer, ForeignKey(
             "cbsd_states.id", ondelete="CASCADE",
-        ), nullable=False,
+        ), nullable=False, index=True,
```

Comment:
Does this really help? This table has only 2 possible values

Reply:
you're right

Reply:
deleted